consensus/state.go

```diff
@@ -732,6 +731,7 @@ func (cs *ConsensusState) handleTxsAvailable() {
 	cs.mtx.Lock()
 	defer cs.mtx.Unlock()
 	// we only need to do this for round 0
+	cs.enterNewRound(cs.Height, 0)
```

Comment:
any idea why we might not have done this initially?! I guess it was just overlooked :(

Reply:
We spent so many hours trying to address liveness and we missed elephant in the room. This is what I am thinking yesterday whole day. Formal verification to the rescue?